zproject/settings.py

```diff
@@ -451,7 +451,7 @@ def get_config(section: str, key: str, default_value: Optional[Any]=None) -> Opt
     # TODO: Figure out why this is different from the corresponding
     # value in static/js/presence.js.  Also, probably move it out of
     # DEFAULT_SETTINGS, since it likely isn't usefully user-configurable.
-    'OFFLINE_THRESHOLD_SECS': 5 * 60,
+    'OFFLINE_THRESHOLD_SECS': 30,
```

Comment:
I guess you have done this for testing purpose, so I hope you'll remove this when you're finished with tests.

Reply:
Yeah, I'll be updating this too.Thanks!